manager/src/main/java/com/redhat/service/smartevents/manager/observability/ManagerObservabilityHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.smartevents.manager.observability;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.redhat.service.smartevents.infra.observability.BaseObservabilityHandler;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.runtime.Startup;
+
+/**
+ * Initialises Secret required by the Observability Operator in the Control Plane.
+ * Changes to the underlying {@link ConfigProperty} values are handled by the app-interface
+ * deployment process.
+ */
+@Startup()
+@ApplicationScoped
+public class ManagerObservabilityHandler extends BaseObservabilityHandler {
```

Comment:
I might be wrong but I don't think we should use the observability operator for the manager on app-interface. Monitoring/alerting for app-interface should be managed with prometheus/grafana and PagerDuty, not the observability operator afaik

Reply:
Ah, forget it, same comment as rob

Reply:
Interestingly.... the Observability Operator simply sets-up Prometheus/Grafana/PagerDuty etc based on configuration we provide... IDK it was "for free" in `app-interface` :smiley: